### PR TITLE
v10.2.0

### PR DESCRIPTION
**Pull request by rickycodes**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -259,33 +272,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+[3.2.0]: git+https://github.com/MetaMask/controllers/compare/v3.1.0...v3.2.0
+[3.1.0]: git+https://github.com/MetaMask/controllers/compare/v3.0.1...v3.1.0
+[3.0.1]: git+https://github.com/MetaMask/controllers/compare/v3.0.0...v3.0.1
+[3.0.0]: git+https://github.com/MetaMask/controllers/compare/v2.0.5...v3.0.0
+[2.0.5]: git+https://github.com/MetaMask/controllers/compare/v2.0.4...v2.0.5
+[2.0.4]: git+https://github.com/MetaMask/controllers/compare/v2.0.3...v2.0.4
+[2.0.3]: git+https://github.com/MetaMask/controllers/compare/v2.0.2...v2.0.3
+[2.0.2]: git+https://github.com/MetaMask/controllers/compare/v2.0.1...v2.0.2
+[2.0.1]: git+https://github.com/MetaMask/controllers/compare/v2.0.0...v2.0.1
+[2.0.0]: git+https://github.com/MetaMask/controllers/releases/tag/v2.0.0
```
Comment (Gudahtt) [inline CHANGELOG.md:305]: These changes are not desired. This is caused by the `repository` field in `package.json`. Unfortunately `auto-changelog` only supports HTTPS repository URLs, so we've been updating the manifest to use the HTTPS url in all repos to fix this issue.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [10.1.0] - 2021-06-7
+## [10.2.0]
+
```
Comment (Gudahtt) [inline CHANGELOG.md:10]: This newline shouldn't be here. And I think there are newlines missing later as well. See `yarn auto-changelog --validate` to validate this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Added
+- Adding TokenListController to fetch the token list from token services API ([#478](git+https://github.com/MetaMask/controllers/pull/478))
+- Feature/use @ethereumjs/tx ([#489](git+https://github.com/MetaMask/controllers/pull/489))
+- Standardize repository ([#486](git+https://github.com/MetaMask/controllers/pull/486))
+- Bump @metamask/contract-metadata from 1.25.0 to 1.26.0 ([#492](git+https://github.com/MetaMask/controllers/pull/492))
+### Changed
+- Bump @metamask/contract-metadata from 1.26.0 to 1.27.0 ([#501](git+https://github.com/MetaMask/controllers/pull/501))
+- Bump @metamask/auto-changelog from 2.3.0 to 2.4.0 ([#499](git+https://github.com/MetaMask/controllers/pull/499))
+- Bump ws from 5.2.2 to 5.2.3 ([#491](git+https://github.com/MetaMask/controllers/pull/491))
+- Bump glob-parent from 5.1.1 to 5.1.2 ([#490](git+https://github.com/MetaMask/controllers/pull/490))
```
Comment (Gudahtt) [inline CHANGELOG.md:19]: These three entries don't affect consumers of this package, so they can be omitted. Lockfile-only changes usually aren't worth including.
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.2.0] - 2021-06-30
+### Added
+- Adding TokenListController to fetch the token list from token services API ([#478](git+https://github.com/MetaMask/controllers/pull/478))
+- Feature/use @ethereumjs/tx ([#489](git+https://github.com/MetaMask/controllers/pull/489))
```
Comment (Gudahtt) [inline CHANGELOG.md:12]: This is not very human-readable :thinking: What impact does this have on users of the package?

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Standardize repository ([#486](git+https://github.com/MetaMask/controllers/pull/486))
+- Bump @metamask/contract-metadata from 1.25.0 to 1.26.0 ([#492](git+https://github.com/MetaMask/controllers/pull/492))
```
Comment (Gudahtt) [inline CHANGELOG.md:14]: The two contract metadata updates can be consolidated into one entry. They should be in the same category as well.

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.2.0] - 2021-06-30
+### Added
+- Adding TokenListController to fetch the token list from token services API ([#478](git+https://github.com/MetaMask/controllers/pull/478))
+- Feature/use @ethereumjs/tx ([#489](git+https://github.com/MetaMask/controllers/pull/489))
+- Standardize repository ([#486](git+https://github.com/MetaMask/controllers/pull/486))
```
Comment (Gudahtt) [inline CHANGELOG.md:13]: What impact does this have on users of this package?

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.2.0] - 2021-06-30
+### Added
+- Adding TokenListController to fetch the token list from token services API ([#478](git+https://github.com/MetaMask/controllers/pull/478))
```
Comment (Gudahtt) [inline CHANGELOG.md:11]: Changelog entries are generally imperative, so "Adding" isn't in the correct tense here.

File: CHANGELOG.md (hunk 5 above)
Comment (rickycodes) [reply CHANGELOG.md]: done!

File: CHANGELOG.md (hunk 6 above)
Comment (rickycodes) [reply CHANGELOG.md]: at first glance i think none so I am going to remove it

File: CHANGELOG.md (hunk 7 above)
Comment (rickycodes) [reply CHANGELOG.md]: updated to `Add`

File: CHANGELOG.md (hunk 4 above)
Comment (rickycodes) [reply CHANGELOG.md]: added those details in brackets

File: CHANGELOG.md (hunk 3 above)
Comment (rickycodes) [reply CHANGELOG.md]: done.

File: CHANGELOG.md (hunk 4 above)
Comment (Gudahtt) [reply CHANGELOG.md]: This still isn't very human-readable. How would you expect people to interpret "Feature/use @ethereumjs/tx"? Why the "Feature/" prefix that we don't use anywhere else? What is `@ethereumjs/tx`?

Note that there's no reason to leave this matching the commit message. The purpose of this entry is to explain what changed - that's it.

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,7 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-## [10.1.0] - 2021-06-7
+## [10.2.0] - 2021-06-30
+### Added
+- Add TokenListController to fetch the token list from token services API ([#478](https://github.com/MetaMask/controllers/pull/478))
+- Update `@ethereumjs-tx` to `@ethereumjs/tx` and add `@ethereumjs/common` to support EIP1559 compliant transactions ([#489](https://github.com/MetaMask/controllers/pull/489))
+
+### Changed
+- Bump @metamask/contract-metadata from 1.25.0 to 1.26.0 and 1.26.0 to 1.27.0 ([#492](https://github.com/MetaMask/controllers/pull/492),[#501](https://github.com/MetaMask/controllers/pull/501))
+
+## [10.1.0]
```
Comment (Gudahtt) [inline CHANGELOG.md:17]: It looks like the date for this release got lost again somehow

**Comment by Gudahtt** [comment]
>A new version of the package @metamask/controllers (10.2.0) was published at 2021-06-30T19:36:07.027Z from
[redacted]. The shasum of this package was af7e3e63db5f98bc2e2d1eaf7141b7abc359e83d.

**Pull request merged**